frontend/src/components/Project/SiteSelection/SiteSelectionViewer.js

```diff
@@ -130,7 +130,7 @@ const SiteSelectionViewer = () => {
               showErrorMessage.isError === false && (
                 <ImageMap
                   header={
-                    "Looking at a map with the source locations and historical events from Geonet in the 2003-present period."
+                    "Source locations and historical events, 2003-present period. (from GeoNet)"
```

Comment:
@claudio525 Is it legit if we change this to:
`Source locations and historical events (Geonet, 2003-present)`

Reply:
Also, it's probably worth updating this dataset periodically...

Reply:
Yeah looks good to me. Only other thought I have is, do we want to specify where we get the source locations from (i.e. NHM)?

Reply:
Yeah, that's probably a good call too. Actually, that's the active fault db from geonet isn't it? not NHM?

Reply:
Ignore me, that's NHM data.

Reply:
Are these the comment I need to worry about? haha
This is also one of them and this format was specified by Brendon.

Reply:
Yep. We're suggesting changing that to `Source Locations (Lichfield, 2014) and historical events (Geonet, 2003-present)`.